app/models/art_item.rb

```diff
@@ -46,23 +47,38 @@ def has_description
       errors.add(:description, "Can't be blank")
     end
   end
+ 
+  private
 
-  def acceptable_documents
+  def validate_documents
     return unless documents.attached?
-  
-    acceptable_types = [
-      "application/pdf", "text/plain", "image/jpg", 
-      "image/jpeg", "image/png", 
-      "application/vnd.openxmlformats-officedocument.wordprocessingml.document"
-    ]
 
-    documents.each do |doc|
-      unless doc.byte_size <= 20.megabyte
-        errors.add(:documents, "is too big")
+    acceptable_doc_types = ['application/pdf', 'application/msword', 
+                          'application/vnd.openxmlformats-officedocument.wordprocessingml.document', 
+                          'text/plain', 'application/rtf', 'application/vnd.ms-excel', 
+                          'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet']
+
+    documents.each do |document|
+      if document.content_type.in?(acceptable_doc_types)
+        if document.blob.byte_size > 10.megabytes # Adjust the size limit as needed
+          errors.add(:documents, "Document size exceeds the 10MB limit.")
```

Comment:
Please consider mentioning the document file size limit in the front end form.

Reply:
I added the limit